appveyor.yml

```diff
@@ -0,0 +1,31 @@
+# General Configuration
+version: 0.0.{build}
+
+# Environment Configuration
+image: Visual Studio 2017
+cache:
+  - '%LocalAppData%\NuGet\v3-cache'
+
+install:
+  #init and update submodules
+  - git submodule update --recursive --init
+
+
+# Build Configuration
+platform: Any CPU
+configuration: Release
+
+before_build:
+  - nuget restore
+
+build:
+  project: SpeckleDynamo.sln
+  verbosity: minimal
+
+after_build:
+  # zip everything into a zip file containing package folder structure
+  - 7z a SpeckleDynamo-$(appveyor_build_version).zip bin\$(configuration)\$(ProjectName)
```

Comment:
You could try...
```
7z a -r SpeckleDynamo-$(appveyor_build_version).zip %APPVEYOR_BUILD_FOLDER%\SpeckleDynamo\bin\Release\*
```
**EDIT:** I figured out where the `bin/` `extra/` and `pkg.json` folders/files were being created...
```
7z a -r SpeckleDynamo-$(appveyor_build_version).zip %APPVEYOR_BUILD_FOLDER%\SpeckleDynamo\bin\Release\SpeckleDynamo\*
```